<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleConfigTabWidget</class>
 <widget class="QWidget" name="ModuleConfigTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget,
QDialog
{
	background-color: rgb(255, 255, 255);
	font-size: 9pt;
}


#wgtModuleConfigs
{
	min-width: 260px;
	max-width: 260px;
	min-height: 80px;
}


QPushButton
{
	background-color: rgb(220, 220, 220);
	border: none;
	border-radius: 6px;
	min-width: 80px;	
	min-height: 24px;
	font-size: 10pt;
}


QPushButton:hover
{
	border-top: 1px solid black;
	border-left: 1px solid black;
	border-right: 2px outset black;
	border-bottom: 2px outset black;
}


QPushButton:pressed
{
	border: 2px inset black;
}


QLineEdit
{
	border: 1px solid black;
	border-radius: 6px;
}


QComboBox
{
	background-color: rgb(230, 230, 230);
	border-radius: 6px;
	border: 1px solid gray;
	min-width: 120px;
	min-height: 18px;
}


#btnSaveSettings
{
	min-width: 170px;
	padding: 0px 12px;
}


#scrollFixedSensors
{
	min-width: 690px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="IssoEditListPanel" name="wgtModuleConfigs" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbSettings">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Настройки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="gbFixedSensors">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Фиксированные датчики</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QScrollArea" name="scrollFixedSensors">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>692</width>
                 <height>0</height>
                </size>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>673</width>
                  <height>1148</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="leftMargin">
                  <number>4</number>
                 </property>
                 <property name="topMargin">
                  <number>4</number>
                 </property>
                 <property name="rightMargin">
                  <number>4</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <property name="spacing">
                  <number>4</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtSmoke" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>200</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtVibration" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtVoltBattery" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtSensorOpen" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtCO" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>200</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtBtnSecurity" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>200</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtTemp" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtBtnFire" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>200</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtVoltBus" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtVoltSensors" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="0">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtSupplyInternal" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="6" column="1">
                  <widget class="IssoAnalogSensorSetupPanel" name="wgtSupplyExternal" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>330</width>
                     <height>180</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Настройка панели управления модулем</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <item row="0" column="0">
              <widget class="QComboBox" name="cbButtonClass1"/>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cbButtonClass2"/>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="cbButtonClass3"/>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="cbButtonClass4"/>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cbButtonClass5"/>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="cbButtonClass6"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>226</width>
             <height>712</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="wgtLinks" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="spacing">
                <number>12</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGroupBox" name="gbAnalogChains">
                 <property name="title">
                  <string>Неадресные  шлейфы</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout">
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig1" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig2" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig3" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig4" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig5" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig6" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="gbDigitalChains">
                 <property name="title">
                  <string>Адресный шлейф</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtChainConfig7" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="gbRelayClassNames">
                 <property name="title">
                  <string>Назначение классов реле</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtIntRelay1" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtIntRelay2" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtIntRelay3" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtIntRelay4" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtIntRelay5" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtExtRelay1" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtExtRelay2" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtExtRelay3" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtExtRelay4" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtExtRelay5" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QGroupBox" name="gbAutomation">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Сценарии автоматики</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="leftMargin">
                   <number>10</number>
                  </property>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtAlarmScenario" native="true"/>
                  </item>
                  <item>
                   <widget class="IssoCheckableSelectionPanel" name="wgtFailScenario" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnSaveSettings">
          <property name="text">
           <string>Сохранить настройки</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IssoEditListPanel</class>
   <extends>QWidget</extends>
   <header>issoeditlistpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IssoAnalogSensorSetupPanel</class>
   <extends>QWidget</extends>
   <header>issoanalogsensorsetuppanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IssoCheckableSelectionPanel</class>
   <extends>QWidget</extends>
   <header>issocheckableselectionpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
